### ISSUE TITLE
Error in `Makie.plot` of rotated raster

### ISSUE THREAD

**Issue by joshday** [rafaqz/Rasters.jl#701]
`Plots.plot` works, but `Makie.plot` produces an error.

## Reproducing

- Use the first download link [here](https://www.sandia.gov/radar/pathfinder-radar-isr-and-synthetic-aperture-radar-sar-systems/complex-data/).  WARNING: 1.29GB download.
- First file in the zipped directory (20151027_1027X04_PS0010_PT000001_N03_M1_CH0_OSAPF.ntf).  I can share this specific file (~90MB) to make reproducing easier.

```Julia
using ArchGDAL, CoordinateTransformations, Rasters, GLMakie

path = "path/to/FARAD_X_BAND/20151027_1027X04_PS0010_PT000001_N03_M1_CH0_OSAPF.ntf"

r = Raster(path)

plot(r)
# ERROR: MethodError: no method matching order(::Rasters.AffineProjected{Int64, CoordinateTransformations.AffineMap{…}, Base.OneTo{…}, DimensionalData.Dimensions.Lookups.NoMetadata, WellKnownText{…}, Nothing, Base.OneTo{…}, X{…}})
```

**Comment by joshday** [comment]
As a temporary workaround, is there a way to remove the coordinate transformation?

**Comment by rafaqz** [comment]
`Makie.heatmap(parent(raster))` !

But we do have the code for the rotation in the plots recipes, it should be too hard to hook up, it's just about time to do it when I don't ever need it.

**Comment by rafaqz** [comment]
Maybe `set(A, X => No lookup, Y => NoLookup)` is better to keep dim names.

On main of DD it's just `set(A, NoLookup)`

**Comment by asinghvi17** [comment]
It could always decompose to matrix like lookups, I suppose...

**Comment by rafaqz** [comment]
No, I never wrote it. Transformed gets you half way. 

But what would that give us for plotting that the affine transform function doesn't? Can we just use the affine transform directly in Makie to transform pixel coordinates?

**Comment by asinghvi17** [comment]
Not really, unless you return a plotspec.  Plus, I don't think it would play well with GeoMakie at all since affine transforms are applied after the nonlinear transform function...

Proj does have the capability to affine transform internally though, so maybe we could use that?  It sounds fragile though.

**Comment by rafaqz** [comment]
Hmm. Making GeoMakie handle affine transforms seems important too. Or is a matrix easier?

**Comment by asinghvi17** [comment]
Well Makie in general can't handle affine transforms before the transform_func, unless you explicitly compose the transform_func. 

A general solution here would be a chained Proj pipeline + `surface` though.
